### ISSUE TITLE
How to train full resolution dataset

### ISSUE THREAD

**Issue by githubfragments** [DeepSceneSeg/AdapNet-pp#11]
Thanks for your nice work. 
I got two questions.
(1) When training the crop of 768x768, what is the batch size for AdapNet and SSMA? Do you use one GPU or multiple GPUs? If multiple GPUs, need synchronized batch normalization?
(2) What's the full resolution used for training ScanNet? The rgb images' resolution is 1296x968, and depth images' resolution is 640x480. What are the sizes you used to train AdapNet_pp and SSMA.

**Comment by avalada** [comment]
(1) We train with a batch size of 16 and use multiple GPUs with synchronized batch normalization.
(2) We train with an input resolution of 480x640 pixels and we resize the predictions to 1296x968 pixels for benchmarking.

**Comment by githubfragments** [comment]
Thanks for your reply.
